src/projects/entities/project.entity.ts

```diff
@@ -24,13 +29,38 @@ export default class Project {
     @JoinColumn()
     owner: User;
 
-    @OneToMany(() => TimeSlot, (timeSlot) => timeSlot.project, {
+    @OneToMany(() => Subject, (subject) => subject.project, {
         onDelete: 'CASCADE',
     })
-    timeSlots: TimeSlot[];
+    subjects: Subject[];
 
-    @OneToMany(() => Subject, (subject) => subject.project, {
+    @OneToMany(() => Department, (department) => department.project, {
         onDelete: 'CASCADE',
     })
-    subjects: Subject[];
+    departments: Department[];
+
+    @OneToMany(() => Schedule, (schedule) => schedule.project, {
+        onDelete: 'CASCADE',
+    })
+    schedules: Schedule[];
+
+    @OneToMany(() => Faculty, (faculty) => faculty.project, {
+        onDelete: 'CASCADE',
+    })
+    faculties: Faculty[];
+
+    @OneToMany(() => RoomType, (roomType) => roomType.project, {
+        onDelete: 'CASCADE',
+    })
+    room_types: RoomType[];
```

Comment:
Use refactoring to fix naming convention violations.

```suggestion
    roomTypes: RoomType[];
```